components/ModFacilitySearchbar.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+    <div
+        class="flex flex-col gap-1 w-fit"
+    >
+        <div
+            class="flex items-center px-3 py-3.5 w-96 h-12 bg-secondary-bg
+            rounded-lg border border-primary-text-muted text-primary-text
+            text-sm font-normal font-sans placeholder-primary-text-muted
+            outline outline-0 -outline-offset-2 gap-2 divide-x-2
```

Comment:
there's an extra `-` in this line

Reply:
-outline-offset-2
![image](https://github.com/user-attachments/assets/387233ee-d765-47c3-aec2-446ef31ff847)

outline-offset-2
![image](https://github.com/user-attachments/assets/95cca2c7-7680-4678-ad61-562c82a0342c)

Should I remove it?

Reply:
ohh sorry, this is correct for a negative value! I didn't realize this was a feature for negative values! (https://tailwindcss.com/docs/upgrade-guide#negative-values-removed)